profiles/fscloud/main.tf

```diff
@@ -0,0 +1,20 @@
+module "rabbitmq_database" {
+  source                     = "../../"
+  resource_group_id          = var.resource_group_id
+  instance_name              = var.instance_name
+  plan_validation            = var.plan_validation
+  region                     = var.region
+  rabbitmq_version           = var.rabbitmq_version
+  endpoints                  = var.endpoints
+  tags                       = var.tags
+  existing_kms_instance_guid = var.existing_kms_instance_guid
+  service_credential_names   = var.service_credential_names
+  kms_key_crn                = var.kms_key_crn
+  admin_pass                 = var.admin_pass
+  members                    = var.members
+  users                      = var.users
+  disk_mb                    = var.disk_mb
+  cpu_count                  = var.cpu_count
+  auto_scaling               = var.auto_scaling
+  cbr_rules                  = var.cbr_rules
```

Comment:
This is missing `kms_encryption_enabled = true` so hence pipeline is failing

Reply:
added